Library/ViewModels/CommentCellViewModel.swift

```diff
@@ -68,6 +74,13 @@ public final class CommentCellViewModel:
       badge.takeWhen(self.bindStylesProperty.signal)
     )
 
+    self.flagButtonIsHidden = self.commentAndProject.signal
+      .ignoreValues()
```

Comment:
`.ignoreValues()` is redundant here is we're doing `.map { _ ... }` on the next line.

Reply:
Removed it and renamed the feature flag.